<bug>
          <bug_id>5578</bug_id>
          
          <creation_ts>2011-09-01 17:50:00 +0200</creation_ts>
          <short_desc>QxSelenium.getRunInContext does not work in FF6</short_desc>
          <delta_ts>2014-03-03 15:25:12 +0100</delta_ts>
          <reporter_accessible>1</reporter_accessible>
          <cclist_accessible>1</cclist_accessible>
          <classification_id>1</classification_id>
          <classification>Unclassified</classification>
          <product>framework</product>
          <component>tool.test.automation</component>
          <version>master</version>
          <rep_platform>Firefox 6.0</rep_platform>
          <op_sys>All</op_sys>
          <bug_status>CLOSED</bug_status>
          <resolution>FIXED</resolution>
          
          
          
          
          
          <priority>P2</priority>
          <bug_severity>major</bug_severity>
          <target_milestone>1.5.1</target_milestone>
          
          
          
          <everconfirmed>1</everconfirmed>
          <reporter name="Daniel Wagner">daniel.wagner</reporter>
          <assigned_to name="Daniel Wagner">daniel.wagner</assigned_to>
          
          <qa_contact name="Daniel Wagner">daniel.wagner</qa_contact>
          <cf_browser>Firefox</cf_browser>
          <cf_browser_version>6</cf_browser_version>

      

      

      

          <long_desc isprivate="0">
            <commentid>26660</commentid>
            <who name="Daniel Wagner">daniel.wagner</who>
            <bug_when>2011-09-01 17:50:25 +0200</bug_when>
            <thetext>QxSelenium.getRunInContext fails with an exception in Firefox 6:
Expected value to be typeof function but found !</thetext>
          </long_desc>
          <long_desc isprivate="0">
            <commentid>26661</commentid>
            <who name="Daniel Wagner">daniel.wagner</who>
            <bug_when>2011-09-01 17:58:27 +0200</bug_when>
            <thetext>Using the AUT window&#39;s Function object has fixed this problem, but not for all use cases: Several Simulator tests still fail, including simulatortest.simulation.QxCommands:test qxClick Coordinates</thetext>
          </long_desc>
          <long_desc isprivate="0">
            <commentid>26701</commentid>
            <who name="Daniel Wagner">daniel.wagner</who>
            <bug_when>2011-09-07 16:02:21 +0200</bug_when>
            <thetext>The test failures are caused by simulator.MEventSupport.addListener, which has the same problem: It takes a callback function which is added to the AUT window via _addOwnFunction and attached to a qx event listener. _addOwnFunction uses getEval, which is executed in the Selenium window&#39;s context. In the AUT context, qx.Bootstrap.isFunction returns false for this function since Object.prototype.toString.call(func) returns &quot;[object Object]&quot; instead of &quot;[object Function]&quot;.</thetext>
          </long_desc>
          <long_desc isprivate="0">
            <commentid>27571</commentid>
            <who name="Daniel Wagner">daniel.wagner</who>
            <bug_when>2011-11-17 16:50:15 +0100</bug_when>
            <thetext>The callback functions used by addListener are now defined by instantiating the AUT windows&#39;s Function object.

Fixed with 
933ca225d9554334b955b7b47132400b3bb5c58a on master
933ca225d9554334b955b7b47132400b3bb5c58a on branch_1_5_x</thetext>
          </long_desc>
          <long_desc isprivate="0">
            <commentid>38662</commentid>
            <who name="Martin Wittemann">martin.wittemann</who>
            <bug_when>2014-03-03 15:25:12 +0100</bug_when>
            <thetext>Closed all bugs already shipped with a release.</thetext>
          </long_desc>
      
      

    </bug>